plume-models/src/migrations.rs

```diff
@@ -77,6 +77,7 @@ impl ImportedMigrations {
 
         let latest_migration = conn.latest_run_migration_version()?;
         let latest_id = if let Some(migration) = latest_migration {
+            println!("{:?}", migration);
```

Comment:
is this still needed, or was this just for debugging?

Reply:
I have no idea, @BaptisteGelez ?